src/dbus.cpp

```diff
@@ -582,14 +525,45 @@ bool dbus_manager::init(const std::string& dest)
 
     std::cout << "MANGOHUD: Connected to D-Bus as \"" << m_dbus_ldr.bus_get_unique_name(m_dbus_conn) << "\"." << std::endl;
 
-    m_dest = dest;
-    dbus_list_name_to_owner(*this, m_name_owners);
-
+    dbus_list_name_to_owner();
     connect_to_signals();
+
+    select_active_player();
+    {
+        std::lock_guard<std::mutex> lck(main_metadata.mtx);
+        get_media_player_metadata(main_metadata.meta);
+    }
+
     m_inited = true;
     return true;
 }
 
+bool dbus_manager::select_active_player() {
+    // If the requested player is available, use it
+    if(m_name_owners.count(m_requested_player) > 0) {
+        m_active_player = m_requested_player;
+        std::cerr << "Selecting requested player: " << m_requested_player << "\n";
+        return true;
+    }
+
+    // Else, use any player that is currently playing..
+    for(const auto& entry : m_name_owners) {
+        const auto& name = std::get<0>(entry);
+        metadata meta;
+        get_media_player_metadata(meta, name);
+        if(meta.playing) {
+            m_active_player = name;
```

Comment:
Maybe even `main_metadata.meta = meta;` here, for when audacious etc is paused at startup

Reply:
Just solved this by passing in a pointer to a `metadata` and storing the updated values there.